test/functional/update/update-delta-manifest-invalid.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -L -n "bundle1" -v 10 -f /file1 "$TEST_NAME"
+
+	create_version "$TEST_NAME" 20 10 staging
+	update_bundle -i "$TEST_NAME" bundle1 --update /file1
+
+	# Overwrite delta manifest with invalid file
+	sudo sh -c "echo 1 > $WEBDIR/20/20/Manifest.bundle1.D.10"
```

Comment:
it is ok like tit is now, but just FYI there is a function in the test library to do that:
```
Usage:
    write_to_protected_file [-a] <file> <stream>

Options:
    -a    If used the text will be appended to the file, otherwise it will be overwritten
```